p2p/peering.go

```diff
@@ -119,6 +119,37 @@ type LocalPeer struct {
 	rpMu   sync.Mutex
 }
 
+type SnapShot struct {
+	Id         uint64
+	Services   wire.ServiceFlag
+	InitHeight int32
+	Raddr      string
+	Na         *wire.NetAddress
+	Version    uint32
+	UserAgent  string
+	Banscore   int32
+}
+
+func (rp *RemotePeer) StatsSnapshot() *SnapShot {
```

Comment:
Name this Info and provide a doc comment.

Reply:
done